data/data/powervs/cluster/power_network/pi_network.tf

```diff
@@ -1,7 +1,7 @@
 locals {
-  ids               = data.ibm_pi_dhcps.dhcp_services.servers[*].dhcp_id
-  names             = data.ibm_pi_dhcps.dhcp_services.servers[*].network_name
-  dhcp_id_from_name = var.pvs_network_name == "" ? "" : matchkeys(local.ids, local.names, [var.pvs_network_name])[0]
+  ids                = data.ibm_pi_dhcps.dhcp_services.servers[*].dhcp_id
```

Comment:
these are causing tf-fmt to complain and can probably be squashed anyways since no changes other than whitespace were added.